posthog/management/commands/generate_demo_data.py

```diff
@@ -205,28 +217,40 @@ def print_results(matrix: Matrix, *, seed: str, duration: float, verbosity: int)
         )
         print("\n".join(summary_lines))
 
-    def materialize_common_columns(self) -> None:
+    def materialize_common_columns(self, backfill_days: int) -> None:
         event_properties = {
             *PERSON_PROPERTIES_ADAPTED_FROM_EVENT,
             "$prev_pageview_pathname",
             "$prev_pageview_max_content_percentage",
             "$prev_pageview_max_scroll_percentage",
             "$screen_name",
             "$lib",
+            "$lib_version",
             "$geoip_country_code",
             "$geoip_subdivision_1_code",
             "$geoip_subdivision_1_name",
             "$geoip_city_name",
             "$browser_language",
             "$timezone_offset",
+            "$host",
```

Comment:
I added some extra columns here because the SQL failed if they weren't present. $host is the only one that matters, but I added the rest when I noticed them